book-domain-service/app/db.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+
+from config import Config
+from app import db
+from app import app
```

Comment:
You can import multiple things from the same module in one line.

`from app import app, db`